.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,20 @@
+
+📅 Target dates:
+
+Release Manager: 
+
+Release checklist:
+🚨 Update the release version variable to the version
+you are releasing. 
+
+[ ] `export RELEASE_VERSION=v0.33.1`
+
+[ ] `git tag -sa $RELEASE_VERSION -m "$RELEASE_VERSION" `
+
+[ ] Send release update in the #zarf Kubernetes Channel
```

Comment:
We should add an item to this checklist to merge the PR goreleaser creates on the [homebrew-tap repo](https://github.com/defenseunicorns/homebrew-tap)